### PR TITLE
Update debian.md to ease understanding

### PR DESCRIPTION
**Pull request by i-jonathan**
### Proposed changes

What: Added extra explanation for what should be replaced with for installation on Debian Derivatives like Kali.

Why: To ease understanding of what users should replace the text that should be replaced with.

### CODE REVIEW

File: content/engine/install/debian.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Docker from the repository.
    > ```console
    > $(. /etc/os-release && echo "$VERSION_CODENAME")
    > ```
+   > with the codename for the equivalent version of Debian.
```
Comment (dvdksn) [inline content/engine/install/debian.md:121]:
```suggestion
   >
   > Replace this part with the codename of the corresponding Debian release,
   > such as `bookworm`.
```

**Review by dvdksn — APPROVED** [review]
Thanks ✨

**Comment by i-jonathan** [comment]
Hi @dvdksn 
Just wanted to find out if there's anything else I need to do to get this merged
(sorry if it sounds stupid lol. First time contributing anything)

**Comment by dvdksn** [comment]
Hi @i-jonathan, nothing needed on your part, just me having to remember to press the merge button :) Thanks for helping improve our docs.

**Pull request merged**